tests/test_strategy_external.py

```diff
@@ -128,7 +128,7 @@ def test_accept_offer(self):
         assert response_payload["trade"] == trade.to_JSON_string()
 
     def test_bid(self):
-        payload = {"data": json.dumps({"energy": 12, "price": 32})}
+        payload = {"data": json.dumps({"energy": 12, "price": 32, "transaction_id": "some_id"})}
```

Comment:
General comment, but since this is my major concern, could you also validate with a test that messages without a transaction_id are correctly rejected ? Thanks !

Reply:
I am definitely in favour of testing the external strategies. But I just realise, that this unittest only tests the `ExternalStrategy` which we dont use any more. Right?
If this is true i will revert https://github.com/gridsingularity/d3a/pull/680/files#diff-4227e14dfec828d6e1d346e65d981d94R197 too
I dont know why i added it.
And I will add unittedts for the external strategies ,ok?

Reply:
That would be great yes. I also got confused and thought that we already had unit tests for these. If you do not want to take the time to do it all, just add the register unit test, and add TODO comments so that we deal with these at the first time we need to solve a ticket for each of the strategies.

Reply:
OK! Thanks! Adding the unittest for the register command.